crates/store/src/lib.rs

```diff
@@ -6,17 +6,28 @@
 #![feature(generic_associated_types)]
 #![feature(type_alias_impl_trait)]
 
+mod error;
+#[cfg(not(feature = "memory"))]
+mod filesystem;
+#[cfg(feature = "memory")]
 mod memory;
 
-pub use memory::*;
+pub use error::Error;
+#[cfg(not(feature = "memory"))]
+pub use filesystem::Filesystem as Storage;
+#[cfg(feature = "memory")]
+pub use memory::Memory as Storage;
```

Comment:
@rvolosatovs any opinion on renaming as Storage when I export these? I like being able to swap implementations with a feature flag for sanity testing and better performance in unit tests